TabLayout/tests/TabGroup-specs.js

```diff
@@ -32,15 +34,19 @@ describe('TabGroup specs', () => {
 				]}
 			/>
 		);
-
-		const expected = 3;
-		const actual = subject.find('Tab').length;
-
-		expect(actual).toEqual(expected);
+		const firstTab = screen.getByRole('group').children.item(0);
```

Comment:
Can we use `getByText` instead of this?

Reply:
Or could we just check the length of screen.getByRole('group').children?

Reply:
Done